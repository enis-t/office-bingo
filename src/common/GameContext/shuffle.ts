import randomizeArray from '@/utils/randomizeArray'

import { bingoItem } from '@/types/bingoItem'

const shuffle = ( items: bingoItem[] ) => {
  let finalItems: bingoItem[] = []
  const slicesByPopularity = [ 2, 7, 15 ] // 2 rare, 7 average, 15 frequent

  const ordered = ( items )
    .sort( ( a, b ) => b.frequency - a.frequency )

  slicesByPopularity.forEach( ( amountOfItems, i ) => {
    const sliceSize = Math.floor( ordered.length / slicesByPopularity.length )

    const currentSlice = ordered.slice( i * sliceSize, ( i * sliceSize ) + amountOfItems )

    finalItems = [
      ...finalItems,
      ...randomizeArray( currentSlice ).slice( 0, amountOfItems ),
    ]
  } )

  finalItems = randomizeArray( finalItems )
  finalItems.splice( finalItems.length / 2, 0, {
    frequency: -1,
    id: 'empty',
    phrase: '',
  } )

  return finalItems
}

export default shuffle