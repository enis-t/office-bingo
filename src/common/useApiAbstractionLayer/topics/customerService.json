[
  {
    "frequency": 0.50,
    "id": "50",
    "phrase": "Thank you for contacting [Company Name]. How may I assist you?"
  },
  {
    "frequency": 0.51,
    "id": "51",
    "phrase": "I apologize for any inconvenience caused."
  },
  {
    "frequency": 0.52,
    "id": "52",
    "phrase": "Let me check the status of your order/account."
  },
  {
    "frequency": 0.53,
    "id": "53",
    "phrase": "I'll do my best to resolve this issue for you."
  },
  {
    "frequency": 0.54,
    "id": "54",
    "phrase": "Please provide me with your account details."
  },
  {
    "frequency": 0.55,
    "id": "55",
    "phrase": "I understand your frustration and will help you find a solution."
  },
  {
    "frequency": 0.56,
    "id": "56",
    "phrase": "Could you please provide more information about the issue?"
  },
  {
    "frequency": 0.57,
    "id": "57",
    "phrase": "I'll escalate this matter to our technical support team."
  },
  {
    "frequency": 0.58,
    "id": "58",
    "phrase": "Let's troubleshoot the problem step by step."
  },
  {
    "frequency": 0.59,
    "id": "59",
    "phrase": "I appreciate your patience while I investigate this further."
  },
  {
    "frequency": 0.60,
    "id": "60",
    "phrase": "I'll check if there are any known issues with our system."
  },
  {
    "frequency": 0.61,
    "id": "61",
    "phrase": "Please hold on for a moment while I access your records."
  },
  {
    "frequency": 0.62,
    "id": "62",
    "phrase": "I'm sorry for the misunderstanding. Let me clarify."
  },
  {
    "frequency": 0.63,
    "id": "63",
    "phrase": "I'll make sure to pass along your feedback to the relevant department."
  },
  {
    "frequency": 0.64,
    "id": "64",
    "phrase": "I'll provide you with regular updates on the progress."
  },
  {
    "frequency": 0.65,
    "id": "65",
    "phrase": "Let me offer you a few options to resolve the issue."
  },
  {
    "frequency": 0.66,
    "id": "66",
    "phrase": "I'll forward your request to our billing department."
  },
  {
    "frequency": 0.67,
    "id": "67",
    "phrase": "Please let me know if there's anything else I can assist you with."
  },
  {
    "frequency": 0.68,
    "id": "68",
    "phrase": "I'll double-check the details and get back to you shortly."
  },
  {
    "frequency": 0.69,
    "id": "69",
    "phrase": "I apologize for the delay. Let me expedite the process for you."
  },
  {
    "frequency": 0.70,
    "id": "70",
    "phrase": "I'll review your account history to better understand the situation."
  },
  {
    "frequency": 0.71,
    "id": "71",
    "phrase": "Let's find a suitable solution that meets your needs."
  },
  {
    "frequency": 0.72,
    "id": "72",
    "phrase": "I appreciate your feedback and will take it into consideration."
  },
  {
    "frequency": 0.73,
    "id": "73",
    "phrase": "I'll make sure to address this matter with our team."
  },
  {
    "frequency": 0.74,
    "id": "74",
    "phrase": "Please accept our sincerest apologies for any inconvenience caused."
  },
  {
    "frequency": 0.75,
    "id": "75",
    "phrase": "I'll provide you with step-by-step instructions to resolve the issue."
  },
  {
    "frequency": 0.76,
    "id": "76",
    "phrase": "Let's schedule a callback at your convenience to further assist you."
  },
  {
    "frequency": 0.77,
    "id": "77",
    "phrase": "I'll investigate the issue with our shipping carrier."
  },
  {
    "frequency": 0.78,
    "id": "78",
    "phrase": "I understand the urgency and will prioritize your request."
  },
  {
    "frequency": 0.79,
    "id": "79",
    "phrase": "Please allow me a moment to review our company's policy."
  },
  {
    "frequency": 0.80,
    "id": "80",
    "phrase": "I'll check if we have a similar product/service available."
  },
  {
    "frequency": 0.81,
    "id": "81",
    "phrase": "I apologize for the confusion. Let's clarify the terms for you."
  },
  {
    "frequency": 0.82,
    "id": "82",
    "phrase": "I'll initiate a refund/credit on your account."
  },
  {
    "frequency": 0.83,
    "id": "83",
    "phrase": "Please rest assured that we will do everything possible to resolve this."
  },
  {
    "frequency": 0.84,
    "id": "84",
    "phrase": "I'll ensure that your concerns are shared with our management team."
  },
  {
    "frequency": 0.85,
    "id": "85",
    "phrase": "Let me provide you with the necessary documentation/information."
  },
  {
    "frequency": 0.86,
    "id": "86",
    "phrase": "I apologize for the oversight. Let me rectify it for you."
  },
  {
    "frequency": 0.87,
    "id": "87",
    "phrase": "I'll verify your warranty details and assist accordingly."
  },
  {
    "frequency": 0.88,
    "id": "88",
    "phrase": "Let's troubleshoot together to identify the root cause of the issue."
  },
  {
    "frequency": 0.89,
    "id": "89",
    "phrase": "I'll pass along your compliments to the team member involved."
  },
  {
    "frequency": 0.90,
    "id": "90",
    "phrase": "Please provide me with your contact information for further assistance."
  },
  {
    "frequency": 0.91,
    "id": "91",
    "phrase": "I apologize for the wait. We'll assist you as soon as possible."
  },
  {
    "frequency": 0.92,
    "id": "92",
    "phrase": "Let me transfer you to the appropriate department for further support."
  },
  {
    "frequency": 0.93,
    "id": "93",
    "phrase": "I appreciate your understanding as we work towards a resolution."
  },
  {
    "frequency": 0.94,
    "id": "94",
    "phrase": "I'll make sure to document your feedback for improvement."
  },
  {
    "frequency": 0.95,
    "id": "95",
    "phrase": "Let me explain our return/exchange process to you."
  },
  {
    "frequency": 0.96,
    "id": "96",
    "phrase": "I'll verify if the requested item is in stock for you."
  },
  {
    "frequency": 0.97,
    "id": "97",
    "phrase": "Please keep me updated on any changes or developments."
  },
  {
    "frequency": 0.98,
    "id": "98",
    "phrase": "I'll take ownership of this issue and ensure it's resolved promptly."
  },
  {
    "frequency": 0.99,
    "id": "99",
    "phrase": "Thank you for bringing this to our attention. We'll investigate further."
  }
]