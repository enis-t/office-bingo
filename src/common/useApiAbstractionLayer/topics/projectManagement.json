[
  {
    "frequency": 0.00,
    "id": "100",
    "phrase": "Let's kick off the project with a team meeting."
  },
  {
    "frequency": 0.01,
    "id": "101",
    "phrase": "What are the project objectives and deliverables?"
  },
  {
    "frequency": 0.02,
    "id": "102",
    "phrase": "We need to define the project scope and timeline."
  },
  {
    "frequency": 0.03,
    "id": "103",
    "phrase": "Let's assign roles and responsibilities to team members."
  },
  {
    "frequency": 0.04,
    "id": "104",
    "phrase": "I'll create a project plan outlining the tasks and milestones."
  },
  {
    "frequency": 0.05,
    "id": "105",
    "phrase": "We should conduct a risk assessment for potential issues."
  },
  {
    "frequency": 0.06,
    "id": "106",
    "phrase": "Are there any dependencies we should be aware of?"
  },
  {
    "frequency": 0.07,
    "id": "107",
    "phrase": "Let's schedule regular status meetings to track progress."
  },
  {
    "frequency": 0.08,
    "id": "108",
    "phrase": "I'll send out meeting agendas and minutes for each session."
  },
  {
    "frequency": 0.09,
    "id": "109",
    "phrase": "We need to establish clear communication channels."
  },
  {
    "frequency": 0.10,
    "id": "110",
    "phrase": "Please provide an update on your task's progress."
  },
  {
    "frequency": 0.11,
    "id": "111",
    "phrase": "Do we have the necessary resources for the project?"
  },
  {
    "frequency": 0.12,
    "id": "112",
    "phrase": "I'll prepare a budget and resource allocation plan."
  },
  {
    "frequency": 0.13,
    "id": "113",
    "phrase": "Let's identify key stakeholders and their expectations."
  },
  {
    "frequency": 0.14,
    "id": "114",
    "phrase": "We may need to adjust the project schedule due to unforeseen circumstances."
  },
  {
    "frequency": 0.15,
    "id": "115",
    "phrase": "I'll create a project dashboard to visualize progress."
  },
  {
    "frequency": 0.16,
    "id": "116",
    "phrase": "Are there any potential bottlenecks we should address?"
  },
  {
    "frequency": 0.17,
    "id": "117",
    "phrase": "I'll facilitate a lessons learned session at project completion."
  },
  {
    "frequency": 0.18,
    "id": "118",
    "phrase": "We should document any changes to the project scope."
  },
  {
    "frequency": 0.19,
    "id": "119",
    "phrase": "Let's ensure everyone is aligned with the project objectives."
  },
  {
    "frequency": 0.20,
    "id": "120",
    "phrase": "I'll monitor project risks and implement mitigation strategies."
  },
  {
    "frequency": 0.21,
    "id": "121",
    "phrase": "Are there any constraints we need to consider?"
  },
  {
    "frequency": 0.22,
    "id": "122",
    "phrase": "Let's conduct regular quality checks throughout the project."
  },
  {
    "frequency": 0.23,
    "id": "123",
    "phrase": "I'll share project updates with the stakeholders."
  },
  {
    "frequency": 0.24,
    "id": "124",
    "phrase": "We may need to reassign resources based on priorities."
  },
  {
    "frequency": 0.25,
    "id": "125",
    "phrase": "I'll schedule a project review with the team."
  },
  {
    "frequency": 0.26,
    "id": "126",
    "phrase": "Do we need to obtain client approval for changes?"
  },
  {
    "frequency": 0.27,
    "id": "127",
    "phrase": "Let's conduct a post-implementation review for lessons learned."
  },
  {
    "frequency": 0.28,
    "id": "128",
    "phrase": "I'll provide training sessions for team members, if needed."
  },
  {
    "frequency": 0.29,
    "id": "129",
    "phrase": "We should document and address any project issues."
  },
  {
    "frequency": 0.30,
    "id": "130",
    "phrase": "Are there any milestones that need special attention?"
  },
  {
    "frequency": 0.31,
    "id": "131",
    "phrase": "I'll create a project communication plan for stakeholders."
  },
  {
    "frequency": 0.32,
    "id": "132",
    "phrase": "Let's ensure the project adheres to the established budget."
  },
  {
    "frequency": 0.33,
    "id": "133",
    "phrase": "I'll monitor project progress against the baseline plan."
  },
  {
    "frequency": 0.34,
    "id": "134",
    "phrase": "Are there any dependencies that are causing delays?"
  },
  {
    "frequency": 0.35,
    "id": "135",
    "phrase": "I'll document project requirements and constraints."
  },
  {
    "frequency": 0.36,
    "id": "136",
    "phrase": "Let's establish a change management process."
  },
  {
    "frequency": 0.37,
    "id": "137",
    "phrase": "We need to conduct regular team status updates."
  },
  {
    "frequency": 0.38,
    "id": "138",
    "phrase": "I'll create a project repository for documentation."
  },
  {
    "frequency": 0.39,
    "id": "139",
    "phrase": "Are there any external factors that could impact the project?"
  },
  {
    "frequency": 0.40,
    "id": "140",
    "phrase": "I'll schedule regular project review meetings with the team."
  },
  {
    "frequency": 0.41,
    "id": "141",
    "phrase": "Let's ensure all project documentation is up to date."
  },
  {
    "frequency": 0.42,
    "id": "142",
    "phrase": "I'll provide regular progress reports to the project sponsor."
  },
  {
    "frequency": 0.43,
    "id": "143",
    "phrase": "We should celebrate project milestones and achievements."
  },
  {
    "frequency": 0.44,
    "id": "144",
    "phrase": "I'll conduct a project closure review to evaluate outcomes."
  },
  {
    "frequency": 0.45,
    "id": "145",
    "phrase": "Are there any lessons learned that we can apply to future projects?"
  },
  {
    "frequency": 0.46,
    "id": "146",
    "phrase": "Let's ensure all project deliverables meet quality standards."
  },
  {
    "frequency": 0.47,
    "id": "147",
    "phrase": "I'll update the project schedule to reflect any changes."
  },
  {
    "frequency": 0.48,
    "id": "148",
    "phrase": "We need to manage stakeholder expectations throughout the project."
  },
  {
    "frequency": 0.49,
    "id": "149",
    "phrase": "I'll archive project files and documentation upon completion."
  }
]