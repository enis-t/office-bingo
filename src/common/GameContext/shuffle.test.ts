import shuffle from './shuffle'

import email from '../useApiAbstractionLayer/topics/email.json'
import customerService from '../useApiAbstractionLayer/topics/customerService.json'
import projectManagement from '../useApiAbstractionLayer/topics/projectManagement.json'


describe( 'unit tests for deal', () => {

  vi.mock( '@/utils/randomizeArray', () => ( { default: ( arr: [] ) => arr } ) )
  it( 'return the correct values', () => {
    expect( shuffle( projectManagement ) ).toMatchSnapshot()
    expect( shuffle( customerService ) ).toMatchSnapshot()
    expect( shuffle( email ) ).toMatchSnapshot()
    expect( shuffle( [
      ...Array( 10 ).fill( { frequency: 1 } ),
      ...Array( 10 ).fill( { frequency: 0.5 } ),
      ...Array( 10 ).fill( { frequency: 0 } ),
    ] ) ).toMatchSnapshot()
  } )
} )