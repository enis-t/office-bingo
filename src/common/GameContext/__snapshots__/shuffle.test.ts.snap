// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`unit tests for deal > return the correct values 1`] = `
[
  {
    "frequency": 0.49,
    "id": "149",
    "phrase": "I'll archive project files and documentation upon completion.",
  },
  {
    "frequency": 0.48,
    "id": "148",
    "phrase": "We need to manage stakeholder expectations throughout the project.",
  },
  {
    "frequency": 0.33,
    "id": "133",
    "phrase": "I'll monitor project progress against the baseline plan.",
  },
  {
    "frequency": 0.32,
    "id": "132",
    "phrase": "Let's ensure the project adheres to the established budget.",
  },
  {
    "frequency": 0.31,
    "id": "131",
    "phrase": "I'll create a project communication plan for stakeholders.",
  },
  {
    "frequency": 0.3,
    "id": "130",
    "phrase": "Are there any milestones that need special attention?",
  },
  {
    "frequency": 0.29,
    "id": "129",
    "phrase": "We should document and address any project issues.",
  },
  {
    "frequency": 0.28,
    "id": "128",
    "phrase": "I'll provide training sessions for team members, if needed.",
  },
  {
    "frequency": 0.27,
    "id": "127",
    "phrase": "Let's conduct a post-implementation review for lessons learned.",
  },
  {
    "frequency": 0.17,
    "id": "117",
    "phrase": "I'll facilitate a lessons learned session at project completion.",
  },
  {
    "frequency": 0.16,
    "id": "116",
    "phrase": "Are there any potential bottlenecks we should address?",
  },
  {
    "frequency": 0.15,
    "id": "115",
    "phrase": "I'll create a project dashboard to visualize progress.",
  },
  {
    "frequency": -1,
    "id": "empty",
    "phrase": "",
  },
  {
    "frequency": 0.14,
    "id": "114",
    "phrase": "We may need to adjust the project schedule due to unforeseen circumstances.",
  },
  {
    "frequency": 0.13,
    "id": "113",
    "phrase": "Let's identify key stakeholders and their expectations.",
  },
  {
    "frequency": 0.12,
    "id": "112",
    "phrase": "I'll prepare a budget and resource allocation plan.",
  },
  {
    "frequency": 0.11,
    "id": "111",
    "phrase": "Do we have the necessary resources for the project?",
  },
  {
    "frequency": 0.1,
    "id": "110",
    "phrase": "Please provide an update on your task's progress.",
  },
  {
    "frequency": 0.09,
    "id": "109",
    "phrase": "We need to establish clear communication channels.",
  },
  {
    "frequency": 0.08,
    "id": "108",
    "phrase": "I'll send out meeting agendas and minutes for each session.",
  },
  {
    "frequency": 0.07,
    "id": "107",
    "phrase": "Let's schedule regular status meetings to track progress.",
  },
  {
    "frequency": 0.06,
    "id": "106",
    "phrase": "Are there any dependencies we should be aware of?",
  },
  {
    "frequency": 0.05,
    "id": "105",
    "phrase": "We should conduct a risk assessment for potential issues.",
  },
  {
    "frequency": 0.04,
    "id": "104",
    "phrase": "I'll create a project plan outlining the tasks and milestones.",
  },
  {
    "frequency": 0.03,
    "id": "103",
    "phrase": "Let's assign roles and responsibilities to team members.",
  },
]
`;

exports[`unit tests for deal > return the correct values 2`] = `
[
  {
    "frequency": 0.99,
    "id": "99",
    "phrase": "Thank you for bringing this to our attention. We'll investigate further.",
  },
  {
    "frequency": 0.98,
    "id": "98",
    "phrase": "I'll take ownership of this issue and ensure it's resolved promptly.",
  },
  {
    "frequency": 0.83,
    "id": "83",
    "phrase": "Please rest assured that we will do everything possible to resolve this.",
  },
  {
    "frequency": 0.82,
    "id": "82",
    "phrase": "I'll initiate a refund/credit on your account.",
  },
  {
    "frequency": 0.81,
    "id": "81",
    "phrase": "I apologize for the confusion. Let's clarify the terms for you.",
  },
  {
    "frequency": 0.8,
    "id": "80",
    "phrase": "I'll check if we have a similar product/service available.",
  },
  {
    "frequency": 0.79,
    "id": "79",
    "phrase": "Please allow me a moment to review our company's policy.",
  },
  {
    "frequency": 0.78,
    "id": "78",
    "phrase": "I understand the urgency and will prioritize your request.",
  },
  {
    "frequency": 0.77,
    "id": "77",
    "phrase": "I'll investigate the issue with our shipping carrier.",
  },
  {
    "frequency": 0.67,
    "id": "67",
    "phrase": "Please let me know if there's anything else I can assist you with.",
  },
  {
    "frequency": 0.66,
    "id": "66",
    "phrase": "I'll forward your request to our billing department.",
  },
  {
    "frequency": 0.65,
    "id": "65",
    "phrase": "Let me offer you a few options to resolve the issue.",
  },
  {
    "frequency": -1,
    "id": "empty",
    "phrase": "",
  },
  {
    "frequency": 0.64,
    "id": "64",
    "phrase": "I'll provide you with regular updates on the progress.",
  },
  {
    "frequency": 0.63,
    "id": "63",
    "phrase": "I'll make sure to pass along your feedback to the relevant department.",
  },
  {
    "frequency": 0.62,
    "id": "62",
    "phrase": "I'm sorry for the misunderstanding. Let me clarify.",
  },
  {
    "frequency": 0.61,
    "id": "61",
    "phrase": "Please hold on for a moment while I access your records.",
  },
  {
    "frequency": 0.6,
    "id": "60",
    "phrase": "I'll check if there are any known issues with our system.",
  },
  {
    "frequency": 0.59,
    "id": "59",
    "phrase": "I appreciate your patience while I investigate this further.",
  },
  {
    "frequency": 0.58,
    "id": "58",
    "phrase": "Let's troubleshoot the problem step by step.",
  },
  {
    "frequency": 0.57,
    "id": "57",
    "phrase": "I'll escalate this matter to our technical support team.",
  },
  {
    "frequency": 0.56,
    "id": "56",
    "phrase": "Could you please provide more information about the issue?",
  },
  {
    "frequency": 0.55,
    "id": "55",
    "phrase": "I understand your frustration and will help you find a solution.",
  },
  {
    "frequency": 0.54,
    "id": "54",
    "phrase": "Please provide me with your account details.",
  },
  {
    "frequency": 0.53,
    "id": "53",
    "phrase": "I'll do my best to resolve this issue for you.",
  },
]
`;

exports[`unit tests for deal > return the correct values 3`] = `
[
  {
    "frequency": 0.49,
    "id": "49",
    "phrase": "I'll confirm the details and get back to you soon.",
  },
  {
    "frequency": 0.48,
    "id": "48",
    "phrase": "I appreciate your patience while we work on this matter.",
  },
  {
    "frequency": 0.33,
    "id": "33",
    "phrase": "I'm sorry, but I need more time to gather the information.",
  },
  {
    "frequency": 0.32,
    "id": "32",
    "phrase": "Let's schedule a follow-up meeting to track progress.",
  },
  {
    "frequency": 0.31,
    "id": "31",
    "phrase": "I'll coordinate with [department/team] to address this issue.",
  },
  {
    "frequency": 0.3,
    "id": "30",
    "phrase": "I recommend we involve [name] for their expertise.",
  },
  {
    "frequency": 0.29,
    "id": "29",
    "phrase": "I'll consult with my team and provide an update.",
  },
  {
    "frequency": 0.28,
    "id": "28",
    "phrase": "I'll make the necessary arrangements and get back to you.",
  },
  {
    "frequency": 0.27,
    "id": "27",
    "phrase": "I suggest we set up a conference call to resolve this.",
  },
  {
    "frequency": 0.17,
    "id": "17",
    "phrase": "I apologize for any confusion caused.",
  },
  {
    "frequency": 0.16,
    "id": "16",
    "phrase": "Please see below for the requested information.",
  },
  {
    "frequency": 0.15,
    "id": "15",
    "phrase": "I understand your concern and will address it promptly.",
  },
  {
    "frequency": -1,
    "id": "empty",
    "phrase": "",
  },
  {
    "frequency": 0.14,
    "id": "14",
    "phrase": "I'll take care of it and provide an update soon.",
  },
  {
    "frequency": 0.13,
    "id": "13",
    "phrase": "I'm cc'ing [name] for their information.",
  },
  {
    "frequency": 0.12,
    "id": "12",
    "phrase": "I'm out of the office until [date], and [colleague] will assist you.",
  },
  {
    "frequency": 0.11,
    "id": "11",
    "phrase": "Your feedback is greatly appreciated.",
  },
  {
    "frequency": 0.1,
    "id": "10",
    "phrase": "Can we arrange a phone call to discuss in detail?",
  },
  {
    "frequency": 0.09,
    "id": "9",
    "phrase": "I'm copying [name] for their input on this matter.",
  },
  {
    "frequency": 0.08,
    "id": "8",
    "phrase": "I'll look into it and get back to you shortly.",
  },
  {
    "frequency": 0.07,
    "id": "7",
    "phrase": "Please let me know if you need any further assistance.",
  },
  {
    "frequency": 0.06,
    "id": "6",
    "phrase": "Let's schedule a meeting to discuss further.",
  },
  {
    "frequency": 0.05,
    "id": "5",
    "phrase": "I appreciate your prompt reply.",
  },
  {
    "frequency": 0.04,
    "id": "4",
    "phrase": "I apologize for the delay in my response.",
  },
  {
    "frequency": 0.03,
    "id": "3",
    "phrase": "Could you clarify your question?",
  },
]
`;

exports[`unit tests for deal > return the correct values 4`] = `
[
  {
    "frequency": 1,
  },
  {
    "frequency": 1,
  },
  {
    "frequency": 0.5,
  },
  {
    "frequency": 0.5,
  },
  {
    "frequency": 0.5,
  },
  {
    "frequency": 0.5,
  },
  {
    "frequency": 0.5,
  },
  {
    "frequency": 0.5,
  },
  {
    "frequency": 0.5,
  },
  {
    "frequency": -1,
    "id": "empty",
    "phrase": "",
  },
  {
    "frequency": 0,
  },
  {
    "frequency": 0,
  },
  {
    "frequency": 0,
  },
  {
    "frequency": 0,
  },
  {
    "frequency": 0,
  },
  {
    "frequency": 0,
  },
  {
    "frequency": 0,
  },
  {
    "frequency": 0,
  },
  {
    "frequency": 0,
  },
  {
    "frequency": 0,
  },
]
`;
